<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FramePlaybackWindow</class>
 <widget class="QDialog" name="FramePlaybackWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Playback</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnStepBack">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/icons/images/NavBack.png</normaloff>:/icons/images/NavBack.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPause">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/icons/images/PauseHS.png</normaloff>:/icons/images/PauseHS.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPlayReverse">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/icons/images/ReverseHS.png</normaloff>:/icons/images/ReverseHS.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStop">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/icons/images/StopHS.png</normaloff>:/icons/images/StopHS.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPlay">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/icons/images/PlayHS.png</normaloff>:/icons/images/PlayHS.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStepForward">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/icons/images/NavForward.png</normaloff>:/icons/images/NavForward.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>25</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QCheckBox" name="cbOriginalTiming">
     <property name="text">
      <string>Use original frame timing from captured frames</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="ckWaitForTraffic">
     <property name="text">
      <string>Wait for traffic before starting playback</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Playback speed (ms):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinPlaySpeed">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Burst Rate:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBurstSpeed">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>8</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Current Playback Item:</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="lblCurrPlayback">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Current frame:</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="lblPosition">
         <property name="text">
          <string>x of y</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QCheckBox" name="cbLoop">
     <property name="text">
      <string>Loop Sequence</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Send to CANBus</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboCANBus"/>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Playback Sequence:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tblSequence"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QPushButton" name="btnLoadFile">
           <property name="text">
            <string>Load File</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnLoadLive">
           <property name="text">
            <string>Load Captured Data</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDelete">
           <property name="text">
            <string>Delete Selected</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>ID Filtering:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listID"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QPushButton" name="btnSelectAll">
           <property name="text">
            <string>Select all IDs</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSelectNone">
           <property name="text">
            <string>Deselect all IDs</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QPushButton" name="btnLoadFilters">
           <property name="text">
            <string>Load Filters</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSaveFilters">
           <property name="text">
            <string>Save Filters</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btnStepBack</tabstop>
  <tabstop>btnPause</tabstop>
  <tabstop>btnPlayReverse</tabstop>
  <tabstop>btnStop</tabstop>
  <tabstop>btnPlay</tabstop>
  <tabstop>btnStepForward</tabstop>
  <tabstop>spinPlaySpeed</tabstop>
  <tabstop>spinBurstSpeed</tabstop>
  <tabstop>cbLoop</tabstop>
  <tabstop>comboCANBus</tabstop>
  <tabstop>tblSequence</tabstop>
  <tabstop>btnLoadFile</tabstop>
  <tabstop>btnLoadLive</tabstop>
  <tabstop>btnDelete</tabstop>
  <tabstop>listID</tabstop>
  <tabstop>btnSelectAll</tabstop>
  <tabstop>btnSelectNone</tabstop>
  <tabstop>btnLoadFilters</tabstop>
  <tabstop>btnSaveFilters</tabstop>
 </tabstops>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
